.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -11,8 +11,7 @@ assignees: ''
 Thank you for contributing to the Dataverse Project through the creation of a feature request!
```

Comment:
Above, should we change

```
title: 'Feature Request/Idea:'
```

to just

```
title: 'Feature Request:'
```

? That is, remove "idea" now that we have a dedicate template for ideas?

Reply:
Nice catch! I removed "Idea"